standalone/src/models/associate-role/fields-config.ts

```diff
@@ -0,0 +1,67 @@
+import { fake, sequence, type TModelFieldsConfig } from '@/core';
+import { ClientLogging } from '@/models/commons';
+import { createRelatedDates } from '@/utils';
+import type { TAssociateRoleGraphql, TAssociateRoleRest } from './types';
+
+const [getOlderDate, getNewerDate] = createRelatedDates();
+
+// buyerAssignable: Scalars['Boolean']['output'];
```

Comment:
I really like adding the expected types as a comment, it makes understanding what's required here more easy to understand at-a-glance